src/dataman/ResultSchemaProvider.cpp

```diff
@@ -129,5 +130,9 @@ nebula::cpp2::Schema ResultSchemaProvider::toSchema() const {
     return schema;
 }
 
+const cpp2::SchemaProp ResultSchemaProvider::getProp() const {
```

Comment:
For ResultSchemaProvider,  we don't need schemaProp_

Reply:
Yeah.
Logically， don't need schemaProp_ in ResultSchemaProvider
Only add  schemaProp_  to NebulaSchemaProvider.
  
Here add  schemaProp_  in  ResultSchemaProvider  only  for unit test.
For example，  `genEdgeSchemaProvider `and `genTagSchemaProvider `in TestUtils.h in storage   use ResultSchemaProvider to  create schema.

Reply:
According to offline communication,   use  NebulaSchemaProvider  to replace ResultSchemaProvider in storage UT.